lang/python/setup_pip.py

```diff
@@ -67,23 +67,23 @@ def build_commands(commands, build_dir, build_env):
 #   Make a quick check of any needed library dependencies, and
 # add to the library path and include path as needed.  If a library
 # is not found, it is not definitive.
-def check_needed_dependencies(builtins, inc_paths, lib_paths):
-    library_dirs = get_library_dirs()
-    compiler = distutils.ccompiler.new_compiler()
-    distutils.sysconfig.customize_compiler(compiler)
-    compiler.set_library_dirs(library_dirs)
-    missing = []
-    for _, libname, instructions in builtins:
-        found = compiler.find_library_file(library_dirs, libname)
-        if found is None:
-            msg(libname + ": missing")
-            msg(instructions)
-            msg("after installing it, set CMAKE_LIBRARY_PATH")
-            missing.append(libname)
-        else:
-            package_top = os.path.dirname(os.path.dirname(found))
-            inc_paths.append(os.path.join(package_top, 'include'))
-            lib_paths.append(os.path.join(package_top, 'lib'))
+# def check_needed_dependencies(builtins, inc_paths, lib_paths):
+#     library_dirs = get_library_dirs()
+#     compiler = distutils.ccompiler.new_compiler()
+#     distutils.sysconfig.customize_compiler(compiler)
+#     compiler.set_library_dirs(library_dirs)
+#     missing = []
+#     for _, libname, instructions in builtins:
+#         found = compiler.find_library_file(library_dirs, libname)
+#         if found is None:
+#             msg(libname + ": missing")
+#             msg(instructions)
+#             msg("after installing it, set CMAKE_LIBRARY_PATH")
+#             missing.append(libname)
+#         else:
+#             package_top = os.path.dirname(os.path.dirname(found))
+#             inc_paths.append(os.path.join(package_top, 'include'))
+#             lib_paths.append(os.path.join(package_top, 'lib'))
```

Comment:
I think we should remove the lines entirely, rather than comment out.  The old lines will live in old versions within the git repository, so we can always go back.

Reply:
removed in most recent commit: [924e8e7](https://github.com/wiredtiger/wiredtiger/pull/10492/commits/924e8e795d33874fa216d570580193aa91958644)